stumpy/gpu_aamp.py

```diff
@@ -47,8 +47,8 @@ def _compute_and_update_PI_kernel(
 
     Parameters
     ----------
-    i : int
-        sliding window `i`
+    idx : int
+        The index for sliding window `i`
```

Comment:
I think the description should say `idx`

```suggestion
        The index for sliding window `idx`
```